frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -12,6 +12,7 @@ const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
             {/* change $10 to info from fixture */}
             <Card.Title>Market Cow Price: ${commons?.cowPrice}</Card.Title>
             <Card.Title>Number of Cows: {userCommons.numOfCows}</Card.Title>
+            <Card.Title>Current Milk Price: ${commons?.milkPrice} </Card.Title>
```

Comment:
It appears the the storybook component for ManageCows needs to be updated to support this.  Right now it looks like this:

<img width="898" alt="image" src="https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3/assets/1119017/495db244-941c-415f-a067-64679e77b9c4">

follow [link](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/prs/19/storybook/?path=/story/components-commons-managecows--uncontrolled) to see for yourself

I'll outline the fix in the next comment.